docs/DeveloperGuide.md

```diff
@@ -297,7 +297,7 @@ _{More to be added}_
 
 **MSS**
 
-1.  Actor requests to <u>list persons (UC2)</u>.
+1.  Actor requests to <u>list persons (UC No.)</u>.
```

Comment:
I think need to update this to `UC2` since the old one already has the UC numbering for list persons?